code/game/objects/obj_defense.dm

```diff
@@ -255,7 +256,7 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 	qdel(src)
 
 //what happens when the obj's health is below integrity_failure level.
-/obj/proc/obj_break(damage_flag)
+/obj/proc/obj_break	(damage_flag)
```

Comment:
?

Reply:
that's called alt tab but you missclick alt

Reply:
```suggestion
/obj/proc/obj_break(damage_flag)
```